lib/dynflow/rails/persistence.rb

```diff
@@ -0,0 +1,26 @@
+module Dynflow
```

Comment:
Indeed, I'll rewrite it in Foreman-tasks so it's Configuration file overrides initialize_persistence with one that actually saves stuff like this.